model/task/task.go

```diff
@@ -2155,7 +1961,8 @@ func resetTaskUpdate(t *Task) bson.M {
 		t.TimeTaken = 0
 		t.LastHeartbeat = utility.ZeroTime
 		t.Details = apimodels.TaskEndDetail{}
-		t.HasCedarResults = false
```

Comment:
Ah my thought process was that we do not care about this field for new tasks, but since we still use it for older tasks it makes sense to clear the field if/when the task is restarted. Good catch!

Reply:
Done.